
This patch will skip the spam filtering for messages sent to the SMTP
server on a SMTP AUTHed connection.

From: 	Steven Haigh <netwiz@crc.id.au>

diff -Naur spamass-milter-0.3.1.orig/spamass-milter.cpp spamass-milter-0.3.1/spamass-milter.cpp
--- spamass-milter-0.3.1.orig/spamass-milter.cpp	2006-03-23 22:41:36.000000000 +0100
+++ spamass-milter-0.3.1/spamass-milter.cpp	2010-03-07 13:55:43.000000000 +0100
@@ -775,6 +775,10 @@
   SpamAssassin* assassin;
   struct context *sctx = (struct context *)smfi_getpriv(ctx);
   char *queueid;
+  if (smfi_getsymval (ctx, "{auth_type}") != NULL)
+  {
+    return SMFIS_ACCEPT;
+  }
 
   if (sctx == NULL)
   {
